libsentrykube/kube.py

```diff
@@ -145,21 +146,26 @@ def _consolidate_variables(
           patch the regional override preserving comments.
     """
 
+    service_path = get_service_path(service_name)
+    merge_config = MergeConfig.from_file(f"{service_path}/sentry-kube/merge.yaml")
```

Comment:
We have `<service>/_sk_flags.yaml` - would this fit in there too instead of a separate dir/file?

Reply:
That's the hope of creating a separate directory actually, to move things like that into it